<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BossCtrl</class>
 <widget class="QWidget" name="BossCtrl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BossCtrl</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="addRoomBtn">
     <property name="text">
      <string>添加房间</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="deleteRoomBtn">
     <property name="text">
      <string>删除房间</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="showBillBtn">
       <property name="text">
        <string>显示账单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBillBtn">
       <property name="text">
        <string>关闭账单</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="outputBillBtn">
     <property name="text">
      <string>账单输出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
